documentation/src/main/asciidoc/user_guide/server_protocols_hotrod_wire.adoc

```diff
@@ -129,15 +129,16 @@ When topology-aware or hash-distribution-aware clients will send 0 until they
 have received a reply from the server with the current view id.
 Afterwards, they should send that view id until they receive a new view id
 in a response.
-| Transaction Type    | 1 byte     | This is a 1 byte field, containing one
-of the following well-known supported transaction types (For this version of
-the protocol, the only supported transaction type is 0): +
-+0+ = Non-transactional call, or client does not support transactions.
-The subsequent TX_ID field will be omitted. +
-+1+ = X/Open XA transaction ID (XID). This is a well-known, fixed-size format.
-| Transaction Id      | byte array |  The byte array uniquely identifying the
-transaction associated to this call. Its length is determined by the
-transaction type. If transaction type is 0, no transaction id will be present.
+//TODO this isn't implemented. Implement it will break compatbility with older clients
```

Comment:
there isn't nothing to remove. Decoder10 reads the transactionType but in Decoder2x it don't read it. 
I checked the client code as well and only the Codec13 sends the transactionType. Starting from Codec20, it isn't sent.